tests/tools/kopiarunner/kopia_snapshotter.go

```diff
@@ -289,20 +288,28 @@ func (ks *KopiaSnapshotter) waitUntilServerStarted(ctx context.Context, addr str
 }
 
 // createAndConnectServer creates Repository and a TLS server/client model for interaction
-func (ks *KopiaSnapshotter) createAndConnectServer(serverAddr string, tlsCertFile string, tlsKeyFile string, args ...string) (*exec.Cmd, error) {
+func (ks *KopiaSnapshotter) createAndConnectServer(serverAddr string, args ...string) (*exec.Cmd, error) {
 	var cmd *exec.Cmd
+
 	var err error
+	if err := ks.ConnectOrCreateRepo(args...); err != nil {
+		return nil, err
+	}
 
-	if err = ks.ConnectOrCreateRepo(args...); err != nil {
+	tempDir, err := ioutil.TempDir("", "kopia")
+	if err != nil {
 		return nil, err
 	}
 
-	serverArgs := append([]string{"--tls-generate-cert", "--tls-cert-file", tlsCertFile, "--tls-key-file", tlsKeyFile})
+	tlsCertFile := filepath.Join(tempDir, "kopiaserver.cert")
```

Comment:
This works in terms of getting the jobs to work.
Theses files still need to be cleaned up somehow, otherwise both unit tests and long running jobs will litter.